portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -112,15 +124,72 @@ function insertSearch() {
 
   const searchInput = document.createElement('input');
   searchInput.className = 'form-control form-inline';
+  searchInput.id = 'userSearch';
```

Comment:
@wangtong-g  comment:
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450558433
I would call it "searchQuery"

Reply:
Okay sure I will rename that div.